src/content/protocols/stakewise-v3/ethereum.md

```diff
@@ -0,0 +1,250 @@
+
+These permissions mean that upgrades can directly influence both the handling of user funds and the treatment of unclaimed staking rewards. Logic upgrades to vaults or controllers can modify withdrawal or redemption behaviour. Changes to validator roots or key managers can reassign the staking infrastructure associated with user deposits. Adjustments to liquidation or redemption configuration can affect how OsToken positions settle. Administrative control over fee settings, treasury recipients and reward distribution parameters can alter how yield flows through the system. In multiple cases, an incorrect or hostile upgrade would allow the responsible administrator to redirect assets, obstruct withdrawals or modify redemption terms in ways that affect users’ positions.
+
+> Upgradeability score: High
+
+## Autonomy
+
+StakeWise v3 operates through several external and internal dependencies that influence its behaviour and determine how user funds and rewards are handled. These dependencies include the validator infrastructure managed by privileged entities, the oracle system feeding validator and reward data into the Keeper, and the multi-sig actors controlling key components such as vault implementations, OsToken parameters, the Redeemer, the MerkleDistributor and registry contracts. Because these dependencies provide inputs that the protocol relies on to recognise staking rewards, assign validators, process redemptions and compute vault accounting, their actions directly shape the system’s ongoing operation.
+
+The Keeper contract obtains validator and reward data from a set of approved oracle addresses. The StakeWise DAO multisig determines which oracles participate and how many signatures are required. Any incorrect reporting or coordinated failure of these oracles would affect the accounting of rewards and the system’s view of validator status. Similarly, validator management depends on the keysManager role, the validator root, deposit data registries and the infrastructure operators securing the underlying validators. These actors determine where user stake is deposited and how exited assets are handled. If these dependencies do not behave as expected, user funds or their associated rewards can be misdirected or become inaccessible.
+
+The protocol further depends on the behaviour of the multisigs controlling OsToken, VaultController, VaultsRegistry, the Redeemer and the MerkleDistributor. These entities may update parameters or logic that influence the solvency and redemption mechanics of OsToken, the whitelisting of vault factories and implementations, the composition of redeemable asset baskets and the distribution timing of staking rewards. Their actions therefore form part of the protocol’s operational dependencies and can materially influence how users interact with the system and how their positions evolve.
```

Comment:
As mentioned earlier, this is part of the protocol and not an external depenceny, as such it should be mentioned only in the upgrade section (and the protocol analysis)